src/apprt/gtk/class/surface.zig

```diff
@@ -1227,19 +1228,10 @@ pub const Surface = extern struct {
 
         // Unset environment varies set by snaps if we're running in a snap.
         // This allows Ghostty to further launch additional snaps.
-        if (env.get("SNAP")) |_| {
-            env.remove("SNAP");
-            env.remove("DRIRC_CONFIGDIR");
```

Comment:
Sorry I missed this before, but is removing these other env vars no longer necessary? I believe these were all added because they caused some other issues but just checking.

Reply:
Yeah, @3v1n0 is filtering out all environment variables that could be leaking out of the ghostty snap more dynamically than my static list of env variables. This should be more forward proof and thorough than what I had done.

Reply:
Ah, as Ken said... We're filtering them by their contents, instead of by their name, so the host environment values that are good (as `XDG_CACHE_HOME`, if set) are re-used only if they point to a non-snap location, otherwise we just unset them.

And the idea here is to avoid keep a list updated.